components/org.wso2.carbon.identity.scim2.provider/src/main/java/org/wso2/carbon/identity/scim2/provider/util/SupportUtils.java

```diff
@@ -234,4 +237,22 @@ public static Response buildCreateUserResponse(SCIMResponse scimResponse) {
         }
         return buildResponse(scimResponse);
     }
+
+    /**
+     * This is used to set the flow and initiator in the identity context.
+     *
```

Comment:
Let's update the description or method name to reflect that this can be only used for ADMIN(ADMIN + APPLICATION) flows (Not USER flows)

Reply:
Fixed 2f61995f52ef116225b64e484b280d8914d3ced7